packages/loader/container-loader/src/catchUpMonitor.ts

```diff
@@ -51,12 +55,23 @@ export class CatchUpMonitor implements ICatchUpMonitor {
 		this.opHandler({ sequenceNumber: this.deltaManager.lastSequenceNumber });
 	}
 
-	public disposed: boolean = false;
-	public dispose() {
-		if (this.disposed) {
+	private _disposed: boolean = false;
```

Comment:
Also fixed this issue where the classes `disposed` state was externally mutable. This is not a good pattern.